lib/members_page.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require 'scraped_page'
+
+class MembersPage < ScrapedPage
```

Comment:
Having this be a subclass of ScrapedPage, but not have any fields defined is a bit of a warning sign. Having fields defined in a different class seems even more so.

I suspect things that come from this page should be defined as fields in this class, and then something else should combine them later.